sormas-api/src/test/java/de/symeda/sormas/api/doc/DataDictionaryGenerator.java

```diff
@@ -184,14 +184,6 @@ private XSSFSheet createEntitySheet(XSSFWorkbook workbook, Class<? extends Entit
 			Class<?> fieldType = field.getType();
 			if (fieldType.isEnum()) {
 				// use enum type name - values are added below
-//				Object[] enumValues = fieldType.getEnumConstants();
```

Comment:
Why was this removed?

Reply:
The value list produced by the commented code has been replaced by the enum tables generated by this code in line 255ff:
```
		// enums
		for (Class<Enum<?>> usedEnum : usedEnums) {
			rowNumber = createEnumTable(sheet, rowNumber + 1, usedEnum);
		}
```

My impression was that the commented code just has been forgotten there.
If the decision to implement this change was not final and the intention was indeed to possibly revive the commented code, I'll restore the removed lines.

Reply:
I guess the code is really not needed anymore, so you might as well remove it completely to keep it clean :)

Reply:
The comment `// use enum type name - values are added below` was introduced in the same commit as the new code. It does not refer to the commented code, but to the new solution where the enum type is inserted here instead of its values, and the value table is added below.
The change dates back to November 2018, so I guess chances for the removed code to have been intended for revival are indeed minimal :-)